Hearthstone Deck Tracker/DeckManager.cs

```diff
@@ -537,10 +539,11 @@ public static void UpdateDungeonRunDeck(DungeonInfo info)
 					playerClass = ((CardClass)info.HeroCardClass).ToString().ToUpperInvariant();
 			}
 
-			var deck = DeckList.Instance.Decks.FirstOrDefault(x => x.IsDungeonDeck && x.Class.ToUpperInvariant() == playerClass.ToUpperInvariant()
-																		&& !(x.IsDungeonRunCompleted ?? false)
+			var deck = DeckList.Instance.Decks.FirstOrDefault(x => (!isPVPDR && x.IsDungeonDeck || isPVPDR && x.IsDuelsDeck) && x.Class.ToUpperInvariant() == playerClass.ToUpperInvariant()
+																		&& !((x.IsDungeonRunCompleted ?? false) || (x.IsDuelsRunCompleted ?? false))
 																		&& x.Cards.All(e => cards.Any(c => c.Id == e.Id && c.Count >= e.Count)));
-			if(deck == null && (deck = CreateDungeonDeck(playerClass, cardSet, info.SelectedDeck, loadout)) == null)
+			var baseDbfids = cardSet == CardSet.DARKMOON_FAIRE ? info.DbfIds : info.SelectedDeck;
```

Comment:
Same here again, let's use the `isPVPDR` flag instead to avoid running into set issues later on.